samples/profanityFilter.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+function main(gcsUri) {
+  // [START syncRecognizeWithProfanityFilter]
+  // Filters profanity
+
```

Comment:
I would add a section here that shows users what variables to uncomment:

```
  /**
   * TODO(developer): Uncomment these variables before running the sample.
   */
  // const gcsUri = 'gs://.../';
```

Reply:
Done.